src/apps/beefy/positions.ts

```diff
@@ -75,6 +99,36 @@ const beefyAppTokenDefinition = (
       const { decimals } = tokensByTokenId[tokenId]
       return [toDecimalNumber(BigInt(vault.pricePerFullShare), decimals)]
     },
+    dataProps: {
+      depositTokenId: getTokenId({
```

Comment:
does this only work for beefy single token pools? or will vaults only have a single deposit token?

Reply:
I only validated against a handful single token pools for now, other multiple token vaults also seem to have a single `tokenAddress`, but I am not sure what that exactly means. Investigating them seems out of scope for now as we are only supporting single token pools